README.en.md

```
@@ -0,0 +1,170 @@
+# 🚜 4diac Training: Valve Control & ISOBUS Automation
+
+
+
+[![License](https://img.shields.io/badge/License-EPL%202.0-red.svg)](LICENSE.md)
+[![4diac](https://img.shields.io/badge/Eclipse-4diac-purple)](https://www.eclipse.org/4diac/)
+[![Standard](https://img.shields.io/badge/Standard-IEC%2061131--3-green)](https://en.wikipedia.org/wiki/IEC_61131-3)
+[![Standard](https://img.shields.io/badge/Standard-DIN%20EN%2061131--3-green)](https://de.wikipedia.org/wiki/EN_61131-3)
+[![Standard](https://img.shields.io/badge/Standard-IEC%2061499-green)](https://en.wikipedia.org/wiki/IEC_61499)
+[![Standard](https://img.shields.io/badge/Standard-DIN%20EN%2061499-green)](https://de.wikipedia.org/wiki/EN_61499)
+[![Protocol](https://img.shields.io/badge/Protocol-ISOBUS%20%2F%20J1939-orange)]()
```

markdownlint-cli2 (0.23.2)

[warning] No empty links

(MD042, no-empty-links)